apps/src/StudioApp.js

```diff
@@ -708,6 +708,10 @@ StudioApp.prototype.sortBlocksByVisibility = function(xmlBlocks) {
   return visibleXmlBlocks.concat(hiddenXmlBlocks);
 };
 
+StudioApp.prototype.createModalDialog = function(options) {
+  return this.feedback_.createModalDialog(options);
+};
+
```

Comment:
At this point, does the createModalDialog logic better belong in StudioApp (with feedback then calling back into StudioApp)? Since we're no longer just creating modals for feedback purposes.